src/cockpit/packages.py

```diff
@@ -311,7 +352,10 @@ def serve_checksum(self, channel):
     def serve_file(self, path, channel):
         assert path[0] == '/'
 
-        if self.checksum is not None:
+        # HACK: If the response is language specific, don't cache the file. Caching "po.js" breaks
```

Comment:
I guess we should also do this for the missing-file (ie: English) case above, no?  Otherwise changing to/from English might be broken.

Reply:
Hmm good point! Btw noticed that almost everything is served with no-cache now.